ludwig/models/modules/loss_modules.py

```diff
@@ -324,3 +248,47 @@ def loss_multilabel(logits, vector_labels, loss):
                         'sum': tf.contrib.layers.sum_regularizer,
                         'None': lambda x: None,
                         None: lambda x: None}
+
+
+
+def set_Parameters(num_classes, output_exp, loss):
```

Comment:
The name of this function is not great, it does not explain what the function does. Please change it to `obtain_sampled_values`